lib/_http_client.js

```diff
@@ -68,6 +68,11 @@ function ClientRequest(options, cb) {
   self.socketPath = options.socketPath;
   self.timeout = options.timeout;
 
+  const _method = options.method;
+  if (_method !== undefined && _method !== null
```

Comment:
This can be written as:

```js
if (_method != null && typeof _method !== 'string')
```

If you don't choose to rewrite like that, the `&&` should be moved to the previous line, and the second line should be lined up. I'm surprised the linter doesn't complain.